tests/test_external_search.py

```diff
@@ -1352,9 +1356,9 @@ def populate_works(self):
             lc="n2013008575"
         )
         self.display_name = Contributor(
-            sort_name=Edition.UNKNOWN_AUTHOR, display_name='Ann Leckie'
+            sort_name=Edition.UNKNOWN_AUTHOR, display_name='ann leckie'
         )
-        self.sort_name = Contributor(sort_name='Leckie, Ann')
+        self.sort_name = Contributor(sort_name='LECKIE, ANN')
```

Comment:
I was concerned that without a `lowercase` filter the sort_author_keyword type wouldn't treat these as the same person, so I added a test for it, but it appears to work as is.